### DISCUSSION TITLE
LittleFS

### DISCUSSION THREAD

**Discussion by Grator12** [vvip-68/LedPanelWiFi#43] [Вопросы и ответы] [answered]
При компиляции ArduinoIDE выдает следующие: ... fatal error: LittleFS.h: No such file or directory
 #include <LittleFS.h>            // Формат файловой системы - LittleFS - в ядре 2.0.9
                                                                                                                ^
compilation terminated.

exit status 1

Compilation error: LittleFS.h: No such file or directory
Библиотека скачена. 
![image](https://github.com/vvip-68/LedPanelWiFi/assets/133976358/96328ccd-7d76-4f87-9c38-249ff7c03506)

**Comment by vvip-68** [comment, marked as answer]
Какая библиотека? LittleFS - это часть ядра поддержки микроконтроллера, она уже там внутри.
Если говорит, что файл LittleFS.h не найден - значит у вас ядро какой-то очень древней версии, ибо для ESP8266 она в ядре начиная с версии 2.6.2. Для ESP32 - не помню, но примерно с 2.0.4 кажется.

Читайте рекомендации по используемым версиям ядра и библиотек, настройке скетча в комментариях в начале основного файла скетча - отпадет много вопросов...
